setup.py

```diff
@@ -35,6 +35,7 @@
             ],
         'test': [
             'nose',
+            'openfisca-dummy-country'
```

Comment:
Excactly, the version constraint is probably going to come from the other direction (`dummy_country` references a `core`)